lib/Fmt_odoc.ml

```diff
@@ -103,6 +103,17 @@ let fmt_code_block conf s1 s2 =
 
 let fmt_code_span s = hovbox 0 (wrap "[" "]" (str (escape_brackets s)))
 
+let fmt_math_span s = hovbox 2 (wrap "{m " "}" (str s))
+
+let fmt_math_block s =
+  let lines =
+    List.rev
+    @@ List.drop_while ~f:String.is_empty
+    @@ List.rev_map ~f:String.strip
+    @@ String.split_lines s
+  in
+  hvbox 2 (wrap "{math@;" "@;<0 -2>}" (list lines "@;" str))
```

Comment:
This concatenates all the lines if everything fits on one line. Is it intended ?

Also, it seems to add trailing spaces instead of empty lines.

Reply:
fixed